php/laravel/composer.json

```
@@ -1,8 +1,6 @@
 {
   "require": {
-    "fideloper/proxy": "*",
-    "laravel/framework": "~7.27.0",
-    "laravel/tinker": "*"
+    "laravel/framework": "^8.12"
```

Does this constraint ensure we are using 8.12 ?
```suggestion
    "laravel/framework": "~8.22.0"
```

---

No, this is a SemVer requirement, which will allow any 8.x version, but at least 8.12. It picks 8.22 now, but will upgrade to 8.23 etc (so all minor updates). This is the constraint used in laravel/laravel now.

---

yes, it's intended to keep `8.22`, to be aligned with the doc (a bot will take care of update)

---

Ok but because it's SemVer, any 8.x release should work. So ^8 would perhaps be easier then letting the bot update each time.